test/monarch/mock_monarch.F90

```diff
@@ -2,420 +2,348 @@
+  use camp_util, only : assert_msg, almost_equal, to_string
+  use camp_monarch_interface_2
   use camp_mpi
+  use json_module
+
+#ifdef SOLVE_EBI_IMPORT_CAMP_INPUT
+  ! EBI Solver
+  use module_bsc_chem_data
+  use EXT_HRDATA
+  use EXT_RXCM,                               only : NRXNS, RXLABEL
+  ! KPP Solver
+  use cb05cl_ae5_Initialize,                  only : KPP_Initialize => Initialize
+  use cb05cl_ae5_Model,                       only : KPP_NSPEC => NSPEC, &
+          KPP_STEPMIN => STEPMIN, &
+          KPP_STEPMAX => STEPMAX, &
+          KPP_RTOL => RTOL, &
+          KPP_ATOL => ATOL, &
+          KPP_TIME => TIME, &
+          KPP_C => C, &
+          KPP_RCONST => RCONST, &
+          KPP_Update_RCONST => Update_RCONST, &
+          KPP_INTEGRATE => INTEGRATE, &
+          KPP_SPC_NAMES => SPC_NAMES, &
+          KPP_PHOTO_RATES => PHOTO_RATES, &
+          KPP_TEMP => TEMP, &
+          KPP_PRESS => PRESS, &
+          KPP_SUN => SUN, &
+          KPP_M => M, &
+          KPP_N2 => N2, &
+          KPP_O2 => O2, &
+          KPP_H2 => H2, &
+          KPP_H2O => H2O, &
+          KPP_N2O => N2O, &
+          KPP_CH4 => CH4, &
+          KPP_NVAR => NVAR, &
+          KPP_NREACT => NREACT, &
+          KPP_DT => DT
+  use cb05cl_ae5_Parameters,                  only : KPP_IND_O2 => IND_O2
+  use cb05cl_ae5_Initialize, ONLY: Initialize
+#endif
 
   implicit none
 
-  !> File unit for model run-time messages
-  integer, parameter :: OUTPUT_FILE_UNIT = 6
-  !> File unit for model results
-  integer, parameter :: RESULTS_FILE_UNIT = 7
-  !> File unit for script generation
-  integer, parameter :: SCRIPTS_FILE_UNIT = 8
-  !> File unit for results comparison
-  integer, parameter :: COMPARE_FILE_UNIT = 9
-
-  !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-  ! Parameters for mock MONARCH model !
-  !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-
-  !> Number of total species in mock MONARCH
-  integer, parameter :: NUM_MONARCH_SPEC = 800
-  !> Number of vertical cells in mock MONARCH
-  integer, parameter :: NUM_VERT_CELLS = 3
-  !> Starting W-E cell for camp-chem call
-  integer, parameter :: I_W = 1
-  !> Ending W-E cell for camp-chem call
-  integer, parameter :: I_E = 5
-  !> Starting S-N cell for camp-chem call
-  integer, parameter :: I_S = 1
-  !> Ending S-N cell for camp-chem call
-  integer, parameter :: I_N = 5
-  !> Number of W-E cells in mock MONARCH
-  integer, parameter :: NUM_WE_CELLS = I_E-I_W+1
-  !> Number of S-N cells in mock MONARCH
-  integer, parameter :: NUM_SN_CELLS = I_N-I_S+1
-  !> Starting index for camp-chem species in tracer array
-  integer, parameter :: START_CAMP_ID = 100
-  !> Ending index for camp-chem species in tracer array
-  integer, parameter :: END_CAMP_ID = 650
-  !> Time step (min)
-  real, parameter :: TIME_STEP = 1.6
-  !> Number of time steps to integrate over
-  integer, parameter :: NUM_TIME_STEP = 5
-  !> Index for water vapor in water_conc()
+  integer(kind=i_kind), parameter :: NUM_EBI_SPEC = 72
```

Comment:
why were the comments removed from this file?

Reply:
Same as I commented before: It was easier for me to understand the file. I wanted to just put the minimum and work from there.